src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -277,6 +279,20 @@
      */
     public static final Key<Value<Boolean>> ARMOR_STAND_MARKER = DummyObjectProvider.createExtendedFor(Key.class,"ARMOR_STAND_MARKER");
 
+    /**
+     * Represents the {@link Key} for whether players are prevented from taking
```

Comment:
```suggestion
     * Represents the {@link Key} for whether a {@link Player} can take an
     * {@link ItemStack} out of a specific {@link EquipmentSlot} slot from
     * an {@link ArmorStand}.
```

Reply:
Can swap out the other java doc as well, or propose something better? Just my two cents on the doc for clarity.